node-wasm/src/worker.rs

```diff
@@ -239,7 +239,14 @@ impl NodeWorker {
 pub async fn run_worker(queued_events: Vec<MessageEvent>) -> Result<()> {
     info!("Entered run_worker");
     let (tx, mut rx) = mpsc::channel(WORKER_MESSAGE_SERVER_INCOMING_QUEUE_LENGTH);
-    let events_channel_name = format!("NodeEventChannel-{}", get_crypto()?.random_uuid());
+    let events_channel_name = if !is_safari() {
+        format!("NodeEventChannel-{}", get_crypto()?.random_uuid())
+    } else {
+        // TODO: handle uniqueness in Safari
+        // it doesn't have `crypto` module in global scope
+        // should not be an issue as long as we run shared worker there
+        "NodeEventChannel-1".to_string()
+    };
```

Comment:
How `get_crypto()` behaved here? Did it returned an error?

Reply:
Reflection fails because there is no crypto object in global scope